browser/ui/webui/brave_vpn/vpn_panel_handler.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/webui/brave_vpn/vpn_panel_handler.h"
+
+#include <utility>
+
+VPNPanelHandler::VPNPanelHandler(
+    mojo::PendingReceiver<brave_vpn::mojom::PanelHandler> receiver,
+    ui::MojoBubbleWebUIController* webui_controller,
+    GetWebContentsForTabCallback get_web_contents_for_tab,
+    BraveVpnService* service)
+    : receiver_(this, std::move(receiver)),
+      webui_controller_(webui_controller),
+      get_web_contents_for_tab_(std::move(get_web_contents_for_tab)),
```

Comment:
Just came across this PR, you don't need to copy this if you're not using it.
Same for `GetWebContentsFromTabId` anonymous function in vpn_panel_ui.